packages/react/src/components/Forms/DatePicker/DatePickerAux.tsx

```diff
@@ -87,7 +87,12 @@ const DatePickerAux = forwardRef(
 
     return (
       <I18nProvider locale={locale || currentLocale}>
-        <Field {...props}>
+        <Field
+          {...props}
+          className={classNames({
+            "c__date-picker__range__container": !!optionalClassName,
```

Comment:
I agree with what you mentioned in this [comment](https://github.com/openfun/cunningham/pull/227#issuecomment-1891065474) because this line is breaking the SRP and OC principles imo